chapter_assets/email-templates/email-templates.md

```diff
@@ -0,0 +1,210 @@
+#Email templates
+
+## New Sponsor Information
+Hi XXX,
+
+I'm writing you to give you more detailed information about codebar.io and provide you with details for hosting a workshop.
+
+**About codebar**
+
+Codebar.io is an organization that started in the UK. The aim is to help women and other underrepresented groups in tech learn to program.
```

Comment:
```suggestion
Codebar.io is a non-profit initiative that facilitates the growth of a diverse tech community by running regular programming workshops. The organization started in 2016 the UK and has expanded all over the world since. We have chapters from the US to Australia passing by Europe, Ghana and China!
```